Asignment7/Assignment7.py

```diff
@@ -1057,6 +1010,77 @@ def main():
 
     file_html.close()
 
+    return
+
+
+def main():
+
+    """loading the data"""
+    db_user = "root"
+    db_pass = "root"
+    db_host = "localhost"
+    db_database = "baseball"
+    connect_string = f"mariadb+mariadbconnector://{db_user}:{db_pass}@{db_host}/{db_database}"
+
+    sql_engine = sqlalchemy.create_engine(connect_string)
+
+    query = """
+                    SELECT * FROM baseball_game_stats
+                    """
+
+    df = pandas.read_sql_query(query, sql_engine)
+
+    predictors = ['HT_BA',
```

Comment:
I'm unable to run your python code as the predictors' name doesn't match your final SQL table. I think you forgot to commit your updated SQL code.